tload: add page (#21863)
tldr-pages/tldr: pages/linux/tload.md

# tload

> Show system load average using ASCII art.
> See also: `uptime`, `w`, `top`, `ps`.
> More information: <https://manned.org/tload>.

- Run tload:

`tload`

> - Change the graph scale (recommended value=10 to maintain a 1:1 scale based on the system load percentage):

Reviewer: Lastly, the description is too long. How about this instead?

Reviewer: ```suggestion
- Display a graph with a specific vertical scale (a smaller value represents a larger scale):
```

Reviewer: OR
```suggestion
- Display a graph with a specific vertical scale (e.g. 10 for a 1:1 scale):
```

> - Change the graph [s]cale (recommended value=10 to maintain a 1:1 scale based on the system load percentage):

Reviewer: ```suggestion
- Change the graph scale (recommended value=10 to maintain a 1:1 scale based on the system load percentage):
```

Reviewer: Bracketed mnemonics are only used for short-only options. Since tload supports long flags, they are redundant here.

`tload {{[-s|--scale]}} {{value}}`

- Change the refresh rate (seconds) for the graph:

> `tload {{[-d|--delay]}} {{value}}`

Reviewer: ```suggestion
`tload {{[-d|--delay]}} {{seconds}}`
```

- View a server's load using tload:

> `ssh {{username@host}} -t tload {{[-s|--scale]}} 10 {{[-d|--delay]}} 1`

Reviewer: According to our style guide: "Avoid explaining general UNIX concepts that could apply to any command...". 
I suggest replacing the ssh example with one that shows how to target a specific tty, as it is a native feature of `tload`.

Reviewer: ```suggestion
- Display a graph for a specific terminal:

`tload {{/dev/tty1}}`
```

Author: You're right
